### PR TITLE
upgrade to 1.1.0 and get rid of nodegit

### PR DESCRIPTION
**Pull request by haitlahcen**
This PR upgrade clever-tools to the latest `1.1.0`.

The version `0.9.3` is still accessible with the attribute `v0_9_3`.
One can install it by doing `nix-env -i -f default.nix -A v0_9_3`

I had to fix the patched binary (using a fix given by a similar package of nixpkgs) because of `zeit/pkg` used to bundle the executable.
Funny thing is that some offsets were actually hardcoded in the Javascript.

### CODE REVIEW

File: clever-tools/default.nix
Comment (fmenou) [inline clever-tools/default.nix]: Why is that required?

File: clever-tools/default.nix
Comment (fmenou) [inline clever-tools/default.nix]: Can you remove the sarcasm please?

File: clever-tools/default.nix
Comment (fmenou) [inline clever-tools/default.nix:34]: Do you understand what's going on there and why it's needed?

File: clever-tools/default.nix
Comment (fmenou) [inline clever-tools/default.nix:1]: \o/

**Review by fmenou — COMMENTED** [review]
Few remarks

**Comment by fmenou** [comment]
No need to update the `example.nix` file?

File: clever-tools/default.nix
Comment (haitlahcen) [reply clever-tools/default.nix:34]: The zeit/pkg is using hardcoded offset to call some entrypoint or w/e (I have no clue of why he's doing that, probably because it bundle some plain javascript into the executable). 
Because of patchelf being used to add `rpath` and fix the `interpreter`, we get a shifted binary (a little bigger than the original). And that interfere with the aforementioned `hardcoded offset` used by zeit/pkg.

Comment (fmenou) [reply clever-tools/default.nix:34]: Ok, thanks

**Comment by haitlahcen** [comment]
@fmenou done, what do you think ?

File: CHANGELOG.md
Comment (fmenou) [inline CHANGELOG.md:1]: Thanks for that :+1:

File: clever-tools/default.nix
Comment (haitlahcen) [reply clever-tools/default.nix]: Removed !

**Review by fmenou — COMMENTED** [review]
Let's dogfood it internally before merging it. Otherwise LGTM

**Review by paulrbr-fl — APPROVED** [review]
It's been tested internally I belive @haitlahcen let's merge this in if you feel like it :)

**Pull request merged**
